metadata-protobuf/package.json

```diff
@@ -27,7 +27,8 @@
     "lint": "eslint ./src --ext .ts",
     "checks": "tsc --noEmit --pretty && prettier ./ --check && yarn lint",
     "format": "prettier ./ --write",
-    "prepublish": "yarn build"
+    "prepublishOnly": "npm run prepack",
```

Comment:
According to https://docs.npmjs.com/cli/v8/using-npm/scripts, `prepack` is always executed during `npm publish`, so this seems redundant

Reply:
resolved in 3eba5de